tldw_Server_API/app/core/MCP_unified/command_runtime/executor.py

```
@@ -0,0 +1,222 @@
+        self,
+        *,
+        handlers: Mapping[str, CommandHandler] | CommandHandler,
+        spill_dir: str | Path,
+        spill_threshold_bytes: int = 50_000,
+    ) -> None:
+        self._handlers = handlers
+        self._spill_dir = Path(spill_dir)
+        self._spill_threshold_bytes = max(1, int(spill_threshold_bytes))
+        self._execution_counter = 0
+
+    async def execute(self, chain: CommandChain, context: Any | None = None) -> RawExecutionResult:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `context`

(ANN401)